mantidproject/mantid
# Expose muon analysis 2 to users

**Description of work.**

This makes the Muon Analysis 2 interface publicly visible. This also includes changes to how workspace names are generated for the ADS so Muon Analysis and Frequency Domain Analysis do not use the same workspace names.

**To test:**
  * In both workbench and mantidplot check that you can open both Muon Analysis 2 and Frequency Domain Analysis.
  * Load some data and do a fit and check the names make sense.



Fixes 25778 




---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
